vars/tagDeployment.groovy

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env groovy
 
-def call(String microservice, String aws_profile = "test", String tag = null) {
-  tag = tag ?: gitCommit()
-
-  build job: 'run-tag-and-capture-notes-commit-based',
-    parameters: [
-      string(name: 'ENVIRONMENT', value: aws_profile),
-      string(name: 'COMMIT_HASH', value: tag),
-      string(name: 'SERVICE_TO_TAG', value: microservice)
```

Comment:
i'm not sure how this worked before as the TAG_TYPE is not passed to the job!

Reply:
It is defaulted to `release` here: https://github.com/alphagov/pay-chef/blob/master/cookbooks/pay-ci/templates/default/jenkins_jobs/tag-and-capture-notes-commit-based.groovy.erb#L16